generate_analysis_graphs.py

```diff
@@ -206,12 +254,30 @@
         title="Participants per Episode"
     ).save(f"{output_dir}/participants_per_episode.png", scale_factor=IMG_SCALE_FACTOR)
 
+    log.info("Graphing the demographic distributions...")
+    for demographic, counts in demographic_distributions.items():
+        log.info(f"Graphing the distribution of codes for {demographic}...")
+        altair.Chart(
+            altair.Data(values=[{"code_string_value": code_string_value, "number_of_individuals": number_of_individuals}
+                                for code_string_value, number_of_individuals in counts.items()])
+        ).mark_bar().encode(
```

Comment:
We should add data labels here to make the charts easy to interpret.

Reply:
What do you mean here? The graphs already have axis titles and labels.

Reply:
I meant adding data values for each bar e.g https://altair-viz.github.io/gallery/bar_chart_with_labels.html

Reply:
I don't these are necessary, we probably don't want people caring about whether something is 23 or 26 given the kind of data that AVF uses